docs/UserGuide.md

```diff
@@ -135,22 +135,31 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd student in the application.
 * `find Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
 
-#### Clearing all entries : `clear`
+#### Clearing all students : `clear`
 
-Clears all entries from the application.
+Clears all students from the application.
 
 Format: `clear`
 
-### Managing Classes: [Coming soon...]
+### Managing Classes: [coming soon]
 
 #### Adding a class: `add-class`
 
 Adds a class to the application.
 
-Format: `add-class n/NAME`
+Format: `add-class n/CLASS_NAME`
 
 Examples:
-* `add-class n/CS2013T Tutorial T10`
+* `add-class n/CS2103T Tutorial T10`
+
+#### Listing all students within a class : `list-students`
+
+Shows a list of all students within a particular class in the application.
+
+Format: `list-students /by n/CLASS_NAME`
+
+Examples:
+* `list-students /by n/CS2103T Tutorial T10`
```

Comment:
Not sure if this is the best implementation since the `/by` prefix will have no values associated with it when parsed by the `ArgumentTokenizer`

```suggestion
* `list-students c/CS2103T Tutorial T10`
```